Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml

```diff
@@ -0,0 +1,205 @@
+# Created From Coder Socks as Base for texturing
+# Made By NeptuneZvezda on Discord
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripped WhiteandLightBlue
+  name: Striped white and light blue socks
+  description: Wearing these makes you feel just a little bit lazier
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks//Socks/SocksStripedWhiteandLightBlue.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks//Socks/SocksStripedWhiteandLightBlue.rsi
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripedCaptainBlueandGold
+  name: Striped blue and gold Socks
+  description: Striped Socks that make you feel special enough to boss the heads
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCaptainBlueandGold.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCaptainBlueandGold.rsi
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripedCaptainBlueandWhite
+  name: Striped blue and white socks
+  description: Striped Socks that make you feel special enough to boss the heads
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCaptainBlueandWhite.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCaptainBlueandWhite.rsi
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripedCaptainGoldandBlue
+  name: Striped gold and blue socks
+  description: Striped Socks that make you feel special enough to boss the heads
+  components:
+  - type: Sprite
+    sprite:DeltaV/Clothing/Under/Socks/SocksStripedCaptainGoldandBlue.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCaptainGoldandBlue.rsi
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStrippedCargo
+  name: Cargo striped socks
+  description: These Striped Socks make you feel a little bit stronger
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCargo.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCargo.rsi
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripedCargoLO-Qm
+  name: Logistics striped socks
+  description: Striped Socks that make you feel like becoming an independant Station State
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCargoLO-Qm.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/SocksStripedCargoLO-Qm.rsi
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripedEngieeringAtmos
```

Comment:
"Engieering"

Reply:
will work on this next as will have to change in several sections